### PR TITLE
Rename ElasticSearch instrumentation

### PR DESCRIPTION
**Pull request by utpilla**
Towards #196 

## Changes
- Remove the word "Contrib"

We still need to rename the folder names under `src\` and `test`.

### CODE REVIEW

**Comment by utpilla** [comment]
@ejsmith Could you please review these changes? We intend to remove the word "Contrib" completely from the contrib repo packages. Next steps would be to rename the `src\` and `test\` folders and release a package with the new name `OpenTelemetry.Instrumentation.ElasticSearchClient` on Nuget.

File: test/OpenTelemetry.Contrib.Instrumentation.ElasticsearchClient.Tests/OpenTelemetry.Instrumentation.ElasticsearchClient.Tests.csproj [hunk 1]
```diff
@@ -19,7 +19,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\..\src\OpenTelemetry.Contrib.Instrumentation.ElasticsearchClient\OpenTelemetry.Contrib.Instrumentation.ElasticsearchClient.csproj" />
+    <ProjectReference Include="$(RepoRoot)\src\OpenTelemetry.Contrib.Instrumentation.ElasticsearchClient\OpenTelemetry.Instrumentation.ElasticsearchClient.csproj" />
```
Comment (ejsmith) [inline test/OpenTelemetry.Contrib.Instrumentation.ElasticsearchClient.Tests/OpenTelemetry.Instrumentation.ElasticsearchClient.Tests.csproj:22]: Shouldn't this have `Contrib` removed as well?

Comment (utpilla) [reply test/OpenTelemetry.Contrib.Instrumentation.ElasticsearchClient.Tests/OpenTelemetry.Instrumentation.ElasticsearchClient.Tests.csproj:22]: I was thinking of renaming the folders in a separate PR to make it a little easier to review.

**Review by cijothomas — APPROVED** [review]

File: test/OpenTelemetry.Contrib.Instrumentation.ElasticsearchClient.Tests/OpenTelemetry.Instrumentation.ElasticsearchClient.Tests.csproj (hunk 1 above)
Comment (ejsmith) [reply test/OpenTelemetry.Contrib.Instrumentation.ElasticsearchClient.Tests/OpenTelemetry.Instrumentation.ElasticsearchClient.Tests.csproj:22]: Ok, as far as just changing the namespaces, this looks good to me.

**Review by ejsmith — APPROVED** [review]
LGTM

**Pull request merged**